frontend/src/lib/components/ServiceRequestDetails.svelte

```diff
@@ -64,6 +70,18 @@
 					</div>
 				{/if}
 
+				{#if $user}
```

Comment:
I think you can remove all of the `$user` stuff and it should still work with just the `{#if serviceRequest.priority}`...

Reply:
You're right. My bad. I just took it out.